src/docs/common/snippets/common-graalvm-tracing-agent.adoc

```diff
@@ -0,0 +1,32 @@
+GraalVM provides a https://www.graalvm.org/latest/reference-manual/native-image/metadata/AutomaticMetadataCollection/#tracing-agent[Tracing Agent] to easily gather metadata and prepare configuration files.
+
+:exclude-for-build:maven
```

Comment:
should add maven snippet

Reply:
Agent support from NBT is documented here: https://graalvm.github.io/native-build-tools/latest/maven-plugin.html#agent-support

Reply:
resolved via https://github.com/micronaut-projects/micronaut-guides/pull/1346/commits/0950edcbeee33927b9733e8a063948acab73760b